src/main/java/com/musseukpeople/woorimap/couple/application/InviteCodeService.java

```diff
@@ -31,13 +34,19 @@ public InviteCodeResponse createInviteCode(Long inviterId, LocalDateTime expireD
         return new InviteCodeResponse(inviteCode.getCode());
     }
 
-    public Long getInviterIdByInviteCode(String inviteCode) {
+    public Long getIdByCode(String inviteCode) {
         InviteCode foundInviteCode = inviteCodeRepository.findById(inviteCode)
             .orElseThrow(() -> new NotFoundInviteCodeException(ErrorCode.NOT_FOUND_INVITE_CODE, inviteCode));
         return foundInviteCode.getInviterId();
     }
 
-    public void removeInviteCodeByCreatedCouple(Long inviterId, Long receiverId) {
-        inviteCodeRepository.deleteByTwoMemberId(inviterId, receiverId);
+    public void removeInviteCodeByMembers(List<Member> members) {
+        inviteCodeRepository.deleteByInIds(createMemberIds(members));
+    }
+
+    private List<Long> createMemberIds(List<Member> members) {
+        List<Long> ids = new ArrayList<>();
+        members.forEach(member -> ids.add(member.getId()));
+        return ids;
```

Comment:
```suggestion
    private List<Long> createMemberIds(List<Member> members) {
    return members.stream().map(Member::getId).toList();
```
굳이 메서드로 안만들고 위쪽에서 해줘도 될 것 같아요.